### DISCUSSION TITLE
How to create new migration and how to work with saving and displaying columns of IMAGES in "Okipa/laravel-table" ???

### DISCUSSION THREAD

**Discussion by chegmarco1989** [Okipa/laravel-table#106] [Q&A] [answered]
Hello.

**1 -** With Laravel, to create a new table that is created directly in `database/migrations`, we do:

`php artisan make:migration create_flights_table`

So, I would like to know how to combine (or work with) the new tables created via the `php artisan make:migration` command with this package ???

**Or do you think it is no longer necessary to use the** `php artisan make:migration` **command with the `Okipa/laravel-table` package ???**

**2 -** How to work with saving and displaying columns of IMAGES in `Okipa/laravel-table` ???

Thank you for informing us on each of these two points please.

**Comment by Okipa** [comment, marked as answer]
Hi @chegmarco1989, I'm afraid I didn't understood your question.

What do you want to achieve ? Building a table to display on your app from an Eloquent model ? If so, everything is explained is the documentation. Could you be more specific please ?

This package has nothing to do with building database tables, its only purpose is building and displaying data tables on your interfaces. Think of it as an alternative for data-table for example.

Regarding images, you can render your image HTML from a formatter closure or create your own formatter => https://github.com/Okipa/laravel-table#format-column-values
